pkg/apis/monitoring/v1/alertmanager_types.go

```diff
@@ -343,6 +347,55 @@ type AlertmanagerWebSpec struct {
 	Timeout *uint32 `json:"timeout,omitempty"`
 }
 
+// GlobalSMTPConfig configures global SMTP parameters.
+// See https://prometheus.io/docs/alerting/latest/configuration/#configuration-file
+type GlobalSMTPConfig struct {
+	// The default SMTP From header field.
+	// +optional
+	From *string `json:"from,omitempty"`
+
+	// The default SMTP smarthost used for sending emails, including port number.
+	// Port number usually is 25, or 587 for SMTP over TLS (sometimes referred to as STARTTLS).
+	// Example: smtp.example.org:587
```

Comment:
(suggestion) since the HostPort struct splits the host and port into independent and mandatory fields, we can shorten the description.

```suggestion
	// The default SMTP smarthost used for sending emails.
```

Reply:
🤣 OKay Okay, lol